llama-rs/src/llama.rs

```diff
@@ -426,41 +426,31 @@ impl LlamaModel {
                 } else if tensor_name.contains("layers") {
                     if tensor_name.contains("attention.wo.weight") {
                         0
-                    } else if tensor_name.contains("feed_forward.w2.weight") {
-                        0
                     } else {
-                        1
+                        i32::from(!tensor_name.contains("feed_forward.w2.weight"))
                     }
-                } else if tensor_name.contains("output") {
-                    1
                 } else {
-                    0
+                    i32::from(tensor_name.contains("output"))
```

Comment:
I don't know if this is the right fix for this, honestly split_type should be an enum and the comment already tells us the 2 variants should be Columns and Rows